paramFiles/fan_out_fan_in/conf.json

```diff
@@ -0,0 +1,95 @@
+{
+  "general": {
+    "random_seed": 0,
+    "simulation_name": "fan_out_fan_in",
+    "total_steps": 720,
+    "base_date": "2017-01-01"
+  },
+  "default": {
+    "min_amount": 100,
+    "max_amount": 1000,
```

Comment:
is that for the scatter pattern or overall?

Reply:
This is only for the 'normal' actors. The scatter-gather actors min/max amounts are defined in the alertPatterns.csv 
But I think it is good that you are pointing this out -- the numbers are a bit lowish here. 
I anyways wanted to have a more realistic amount distributions. Currently, it is randomly sampling between min and max. I think something with a big skew would be more appropriate. 
Will do that next.